src/components/CaseDetails/index.tsx

```diff
@@ -69,11 +69,11 @@ const caseFields = [
     label: 'Client Language',
     getValue: (data: CaseListing) => data.languages.join(', '),
   },
-  {
-    label: 'Attorney Experience Level',
-    getValue: (data: CaseListing) =>
-      parseExperience(data.experience_needed) || 'N/A',
-  },
+  // {
```

Comment:
let's remove this since we don't need it anymore

Reply:
sg!